src/stash/src/upgrade/v26_to_v27.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+/// No-op migration for adding RBAC related audit events.
```

Comment:
Do we need to backfill existing stash's that won't have the initial RBAC events?

Reply:
I think it's probably unnecessary. The `ALTER OWNER` will be very difficult to figure out who executed what. We'd have to compare current owners with the user who created the object. For the `REVOKE`S we have no way of figuring out if someone executed a `REVOKE`, and we'll lost all information of the `GRANT` that was executed before the `REVOKE`. For the `GRANT`s, we have no way of knowing who executed the statements, when they executed the statements.

Feel free to push back, I think we could put a bunch of `GRANT`s at the end of the audit log with no user or timestamp.

Reply:
I meant only the initial system grants, not the user ones.

Reply:
I think this will also be difficult to put the initial audit events in the right place in the audit log. I can try and put something together though.

Reply:
Oh yeah the ids have to be in order. Let's just leave it as is without the events.